packages/react/src/drafts/SelectPanel2/SelectPanel.tsx

```diff
@@ -3,7 +3,20 @@ import {SearchIcon, XCircleFillIcon, XIcon, FilterRemoveIcon, AlertIcon, ArrowLe
 import {FocusKeys} from '@primer/behaviors'
 
 import type {ButtonProps, TextInputProps, ActionListProps, LinkProps, CheckboxProps} from '../../index'
-import {Button, IconButton, Heading, Box, Tooltip, TextInput, Spinner, Text, Octicon, Link, Checkbox} from '../../index'
+import {
+  Button,
+  IconButton,
+  Heading,
+  Box,
+  Tooltip,
+  TextInput,
+  Spinner,
+  Text,
+  Octicon,
+  Link,
+  Checkbox,
+  useFormControlForwardedProps,
```

Comment:
Using the hook recently created by this PR: https://github.com/primer/react/pull/3632. This hook is also being used in the `InlineAutocomplete` draft component: https://github.com/primer/react/blob/cb54f4298700d369e616aaefb2e7df4830231736/packages/react/src/drafts/InlineAutocomplete/InlineAutocomplete.tsx#L118